aptos-move/framework/benches/bls12_381.rs

```diff
@@ -0,0 +1,1051 @@
+// Copyright (c) Aptos
```

Comment:
We need to move these into `aptos-crypto` probably and avoid all the `+` stuff like I told you last time.

Reply:
A better name for this file will be `ark_bls12_381.rs`, no?

Reply:
Btw, have you checked if arkworks already offers these benchmarks? Maybe we don't even need to rewrite them...